src/components/pages/home/index.tsx

```diff
@@ -256,6 +256,7 @@ function HomePage({ accountId }: { accountId: string | undefined }) {
         <section className="w-full">
               <Newsletter />
         </section> 
+        <script src="//code.tidio.co/fdylvmddtyb7vzsk5frdt3ncrk6cwobs.js" async></script>
```

Comment:
Why is it a script tag? Please use the library to set it up as the CDN is not acceptable in large scale projects. Also you are exposing your tidio ID